mlrun/runtimes/serving.py

```diff
@@ -322,11 +322,17 @@ def set_tracking(
 
         # Applying model monitoring configurations
         self.spec.track_models = True
-        self.spec.tracking_policy = model_monitoring.TrackingPolicy()
+        self.spec.tracking_policy = ""
```

Comment:
```suggestion
        self.spec.tracking_policy = None
```

Are you using a string in purpose? The `spec.tracking_policy` attribute is expected to be of `TrackingPolicy` type, don't put a string in there (especially since all you want is a default, empty value).